src/db.rs

```diff
@@ -15,7 +17,48 @@ pub type Pool = r2d2::Pool<ConnectionManager>;
 pub type Connection = PooledConnection<ConnectionManager>;
 
 fn make_tls_connector(danger_accept_invalid_certs: bool) -> io::Result<MakeTlsConnector> {
-    let connector = TlsConnector::builder()
+    let key = "CA_ROOT_FILE";
+    let ca_file = match env::var_os(key) {
+        Some(s) => s.into_string().unwrap(),
```

Comment:
Could you please not unwrap values and use `map_err` and `prettify_error` for that?